composer.json

```diff
@@ -25,7 +25,7 @@
   "license": "GP-2.0+",
   "require": {
     "php": ">=5.4.0,<7.1",
-    "typo3/cms-core": ">=7.3.1,<8.0"
+    "typo3/cms-core": "^7.3.1 || ^8.0"
```

Comment:
Since the extension won't work with 8.0-8.3, this should be changed to `|| ^8.4`, right?